### ISSUE TITLE
Admin - approve/block/deactivate new user (1)

### ISSUE THREAD

**Issue by Hils** [joomla/joomla-cms#13492]
### Steps to reproduce the issue
_The user happens to be spam by coincidence_
**receive email:**

> Registration approval required for account of Gabe at ...
> 
> Hello administrator,
> 
> A new user has registered at xxxxx.
> The user has verified their email address and requests that you approve their account.
> This email contains their details:
> 
>   Name :  Gabe
>   email:  doubtocomdi1986@hostcalls.com
>  Username:  supepadmlnlspatop
> 
> You can activate the user by selecting on the link below:
>  http://www.xxxx/users-register?task=registration.activate&token=23bbecad5cde6f443349bb6a55b4c3e3
> 

### Expected result
When clicking on that link, I expect to be taken to user management but instead, clicking on that link auto activates user.

### Actual result
So as this user was unwanted, I had to log in as superadmin and go to ...administrator/index.php?option=com_users&view=users. This user should be blocked but selecting the user and clicking on block has no effect except to produce the message that no user has been blocked.

<img width="908" alt="user" src="https://cloud.githubusercontent.com/assets/736887/21719776/6ba02c62-d417-11e6-8adb-bc4696414a11.png">

_note that the user Gabe was selected before clicking block but is automatically deselected during the clicking block action_

<img width="769" alt="iserblock" src="https://cloud.githubusercontent.com/assets/736887/21719901/037cded6-d418-11e6-822a-b2fe4f85f50e.png">

_note that clicking the green tick in the activate column whilst Gabe is selected, has no effect at all_

### System information (as much as possible)
	Value
 
PHP Built On	Linux uk14.siteground.eu 3.12.18-clouder0 #3 SMP Thu Oct 20 14:54:11 EEST 2016 x86_64
Database Version	5.6.28-76.1-log
Database Collation	utf8_general_ci
Database Connection Collation	utf8mb4_general_ci
PHP Version	5.6.29
Web Server	Apache
WebServer to PHP Interface	cgi-fcgi
Joomla! Version	Joomla! 3.6.5 Stable [ Noether ] 1-December-2016 22:46 GMT  _updated 15 December 2016_
Joomla! Platform Version	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent	Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36

### Additional comments
These are the user options on this site:

<img width="380" alt="useroptions" src="https://cloud.githubusercontent.com/assets/736887/21719985/779eec28-d418-11e6-817c-4b6fe57704b4.png">

**Comment by Hils** [comment]
Further clarification:

The use of the two different words - 'block' which is a button above the user section BUT, as I just found out, also appears as a tool tip when hovering over the enabled column by the user. So rather than using BLOCK and UNBLOCK or ENABLE and DISABLE, we are  confusing the user in my view.

**Comment by frankmayer** [comment]
I agree with @Hils in both counts.

**Comment by PhilETaylor** [comment]
The link CLEARLY STATES

You can **activate the user** by selecting on the link below:
http://www.xxxx/users-register?task=registration.activate&token=23bbecad5cde6f443349bb6a55b4c3e3

But your expectation is:

>When clicking on that link, I expect to be taken to user management but instead, clicking on that link auto activates user.

So I say your expectation is incorrect. Not the link or the code.

**Comment by nibra** [comment]
Please create another issue for the UX issue (block/enable).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13492">issues.joomla.org/tracker/joomla-cms/13492</a>.</sub>
